Cargo.toml

```diff
@@ -25,6 +25,7 @@ aes = { version = "0.8.3", features = ["hazmat"] }
 measureme = "11"
 ctrlc = "3.2.5"
 chrono = { version = "0.4.38", default-features = false, features = ["clock"] }
+chrono-tz = "0.4"
```

Comment:
Why did you add such an ancient version of chrono-tz? The current version is 0.9.

Reply:
Probably because 4 looks like 9.